dist/angular-swaggerific.js

```diff
@@ -129,16 +129,24 @@
         AngularSwaggerific.prototype.trigger = function(path, method, data, config) {
             var self = this;
 
-            var data = data || {};
+            var getParams, postData;
+            if (angular.lowercase(method) === 'get') {
+                getParams = data || {};
+            } else {
+                postData = data || {};
+            }
             var config = config || {};
             var newPath = util.replaceInPath(path, data);
 
             var httpConfig = angular.extend({
                 method: method,
                 url: self.host + newPath,
-                data: data
+				data: postData,
+				params: getParams
             }, config);
 
+            console.debug(httpConfig);
```

Comment:
@darkgoldblade01 

Everything looks good, but can you remove this line? 

Thanks!